frontend/src/components/form/InputText.vue

```diff
@@ -51,7 +53,7 @@ const fieldActive: Ref<boolean> = ref(false);
     />
 
     <small
-      v-if="fieldActive"
+      v-if="fieldActive || helpVisible"
       :id="`${name}-help`"
     >
```

Comment:
We should probably figure out a common functionality across all our form components. Currently all of them except `InputText` show the help by default. This is the only one with multiple diverging methods.